sdk/samples/router/edit_a_router.py

```diff
@@ -0,0 +1,195 @@
+import turing
+import turing.batch
+import turing.batch.config
+import turing.router.config.router_config
+from turing.router.config.route import Route
+from turing.router.config.router_config import RouterConfig
+from turing.router.config.router_version import RouterStatus
+from turing.router.config.resource_request import ResourceRequest
+from turing.router.config.log_config import LogConfig, ResultLoggerType
+from turing.router.config.traffic_rule import TrafficRule, HeaderTrafficRuleCondition
+from turing.router.config.enricher import Enricher
+from turing.router.config.router_ensembler_config import DockerRouterEnsemblerConfig
+from turing.router.config.common.env_var import EnvVar
+from turing.router.config.experiment_config import ExperimentConfig
+
+
+def main(turing_api: str, project: str):
```

Comment:
Since we show all the steps in detail in the create router sample, I wonder if this one can be kept concise.
1. Get an existing router (in reality, this needn't exist but this is only a sample and the user should put in whatever router if they want)
2. Update some property > Deploy it
3. Update some other property  > Save it (i.e., create version only)